recipes/xorg-proto/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+import glob
+import os
+import re
+import yaml
+
+required_conan_version = ">=1.33.0"
+
+
+class XorgProtoConan(ConanFile):
+    name = "xorg-proto"
+    description = "This package provides the headers and specification documents defining " \
+        "the core protocol and (many) extensions for the X Window System."
+    topics = ("conan", "xproto", "header", "specification")
+    license = "X11"
+    homepage = "https://gitlab.freedesktop.org/xorg/proto/xorgproto"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+
+    generators = "pkg_config"
```

Comment:
Is is required? There are no dependencies.

Reply:
Without, the configure script prints a warning about missing `xorg-macros`.

Reply:
I'm starting to think.
The `xorg-macros` defines `AUTOMAKE_CONAN_INCLUDES`, so for cross building, it needs to be a build requirement. Otherwise the environment variable is not visible.

Does the `pkg_config` generator also generate pc files for build requirements? Or do I have to explicitly add `xorg-macros` as an ordinary requirement?

Reply:
from build log:

```
xorg-proto/2021.4: Generator pkg_config created xorg-macros.pc
xorg-proto/2021.4: Generator pkg_config created libpkgconf.pc
```

Reply:
Also when using 2 profiles for cross building?

Reply:
So I have to add `xorg-macros` twice when cross building (both as host and build requirement).